lib/WebService/Client.pm

```diff
@@ -80,20 +80,17 @@ sub delete {
 sub req {
     my ($self, $req) = @_;
     $self->_log_request($req);
-    my $res = $self->ua->request($req);
-    Moo::Role->apply_roles_to_object($res, 'HTTP::Response::Stringable');
```

Comment:
Previously this role would only be applied to the first response. If the request was retried, the new response would not have this role.